intents.yaml

```
@@ -2127,3 +2127,50 @@ HassTimerStatus:
           - start_hours: hours that the timer was started with
           - start_minutes: minutes that the timer was started with
           - start_seconds: seconds that the timer was started with
+
+# -----------------------------------------------------------------------------
+# lawn mower
+# -----------------------------------------------------------------------------
+
+HassLawnMowerStartMowing:
+  supported: true
+  domain: lawn_mower
+  description: "Starts a lawn mower"
+  importance: "complete"
```

```suggestion
  importance: "optional"
```

All new intents are first optional, then if needed required, usable, complete after some time where we communicate with all language leaders


This is new, it gives language leaders a chance to adapt instead of having lots of language going from Complete to usable in a day